### DISCUSSION TITLE
how to use my own image icons for a geojson point layer

### DISCUSSION THREAD

**Discussion by hardreddata** [TerriaJS/terriajs#4901] [Q&A] [answered]
I had a bit of a look through the gitter archives for this.

I was thinking I probably had to use czml but now I wonder if I can use `markerUrl` with a `"type":"geojson"` catalog item.

Advice invited.

Thanks.

**Comment by hardreddata** [comment, marked as answer]
Disregard.

I found https://gist.githubusercontent.com/steve9164/7cd7099b0495875ee0bf1ff651922095/raw/232fc7f73a685f9bab2288709494769335e932cc/test-point-marker-url.json and extend it with `marker-height` and `marker-width`

**Reply by anujdivesh** [reply]
Hi @RussellGrew 
would be have example of adding ```marker-height``` ```marker-width``` Having issues with v8.

**Reply by hardreddata** [reply]
Hi @anujdivesh 

I don't know if this still works in v8.

The v7 example for a catalog item:

```
{
   "name":"My layer",
   "type":"geojson",
   "url":"datapoints.geojson",
   "style":{
      "marker-url":"icon.png",
      "marker-width":30,
      "marker-height":30
   },
   "description":"words here"
}
```

Good luck!
